hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -268,6 +263,32 @@ private void commitOnAutoCommit(String commitTime, JavaRDD<WriteStatus> resultRD
         return records;
     }
 
+    private void saveCommitMetadataToInflight(WorkloadProfile profile, String commitTime) throws HoodieCommitException {
```

Comment:
this is the workload profile - not the commit metadata right?  i.e we still don't know where inserts will go to..

Reply:
Yes, renamed the method for now.